CMakeLists.txt

```
@@ -325,6 +348,10 @@ target_include_directories(uv_a
   PRIVATE
     $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/src>)
 target_link_libraries(uv_a ${uv_libraries})
+set_target_properties(uv_a PROPERTIES OUTPUT_NAME "uv")
```

```suggestion
set_target_properties(uv_a PROPERTIES OUTPUT_NAME "uv_a")
```
Is this what's needed to fix CI?

---

Ah, probably. I see this change was already proposed (and rejected) in the issue someone cross linked above https://github.com/libuv/libuv/pull/2374. So possibly instead should just drop this?